src/Proj4.jl

```diff
@@ -50,14 +48,21 @@ function _get_def(proj::Projection)
 end
 
 """Low level interface to libproj transform, allowing user to specify strides"""
-function _transform!(src::Projection, dest::Projection, point_count, point_stride, x, y, z)
+function _transform!(src::Projection, dest::Projection, point_count::Int64,
+                     point_stride::Int32, x::Ptr{Float64}, y::Ptr{Float64}, z::Ptr{Float64})
```

Comment:
Not specifying the types for `point_count` and `point_stride` was intentional here - as I understand it, `ccall` will convert these to the necessary C types internally, so I thought keeping it generic made some kind of sense: it's perfectly fine to call the function with a `point_count` of Int32 for example.

Actually, the underlying C interface uses `Cint` and `Clong`, which may very well both be `Int32`, even on a 64 bit machine, depending on the platform ABI.